_config.yml

```diff
@@ -25,6 +30,13 @@ defaults:
   values:
     layout: post
     permalink: "/articles/:slug/"
+- scope:
+    path: ''
+    type: 'events'
+  values:
+    layout: events
```

Comment:
Please make the layout 'post' not 'events'. Until we have an 'events' layout made, this will cause an error.

Reply:
This was edited. Collection has now been named properly & sample event was deleted. The events front matter has been made in forestry as well. The 'content type' is a text field, and only available to developers. #34.